packages/material-ui/src/Paper/Paper.d.ts

```diff
@@ -69,6 +71,10 @@ export interface PaperProps extends StandardProps<React.HTMLAttributes<HTMLDivEl
    * @default 'elevation'
    */
   variant?: OverridableStringUnion<PaperVariantDefaults, PaperPropsVariantOverrides>;
+  /**
+   * The system prop that allows defining system overrides as well as additional CSS styles.
+   */
+  sx?: SxProps<Theme>;
```

Comment:
nit: can you add this before variant, so that the props can be sorted?

Reply:
Yes